tests/dump_tests/module_tests/mock_sonicv2connector.py

```diff
@@ -0,0 +1,73 @@
+
+import json, re, os
+from utilities_common.constants import DEFAULT_NAMESPACE
+
+class MockSonicV2Connector():
```

Comment:
Unfortunately no, For writing UT's for the modules, instead of cluttering existing *_db.json files in the mock_tables/* folder, i've decided to have a seperate mock connector and thereby providing the flexibility to have dedicated db files for modules in the dump_tests/files/<module_name_folder>/. 

Another reason being the infra is quite a mess and i couldn't get the mock running on custom dedicated db's made for port module. Hence, for maximum flexibility and ease of use and much cleaner and simpler code, i've decided to implement a MockConnectror itself